libvcx/src/libindy/utils/wallet.rs

```diff
@@ -544,7 +542,7 @@ pub mod tests {
         let id = "id1";
         let value = "value1";
 
-        api_c::vcx::vcx_shutdown(true);
+        // api_c::vcx::vcx_shutdown(true);
```

Comment:
This test is currently in `to_restore` mode, is this change needed?

Reply:
I think the purpose was to remove the only dependence on `api_c` in the file. Not related to the rest of the changes in this PR anyway.